doc/format.md

```diff
@@ -288,20 +356,48 @@ If only one of these features is found in a function, the rule will not match.
 
 # extracted features
 
-capa extracts features from multiple scopes, starting with the most specific (instruction) and working towards the most general:
+capa matches features at multiple scopes, starting small (e.g., instruction) and growing large (e.g., file). In static analysis, scopes grow from instruction, to basic block, function, and then file. In dynamic analysis, scopes from call, to thread, process, and then to file:
 
 | scope       | best for...                                                                              |
 |-------------|------------------------------------------------------------------------------------------|
+| (static)    | ---                                                                                      |
```

Comment:
split this into 2 or 3 tables?

Reply:
[fe77ae4](https://github.com/mandiant/capa-rules/pull/851/commits/fe77ae430dc841f03426b0ed6cc8452128c6cb4f)